README.adoc

```diff
@@ -3,31 +3,27 @@ Raincatcher Documentation
 -------------------------
```

Comment:
This seems to make the header render as an `h2`, the same as the "Overview" title below. I think it needs to render as an `h1`, so `= Raincatcher Documentation` should work I think?